app/utils/report/templates/build.html

```diff
@@ -163,7 +163,7 @@
         {%- set warns = P_("{:d} warning", "{:d} warnings", d.warnings_count).format(d.warnings_count) %}
         {%- set mism = P_("{:d} section mismatch", "{:d} section mismatches", d.mismatches_count).format(d.mismatches_count) %}
         <p>
-    {{ "<strong>{}</strong> ({}) &mdash; {}, {}, {}, {}".format(d.defconfig_full, d.arch, d.status, errs, warns, mism) }}
+    {{ "<strong>{}</strong> ({}, {}) &mdash; {}, {}, {}, {}".format(d.defconfig_full, d.arch, d.compiler_version_ext, d.status, errs, warns, mism) }}
```

Comment:
I keep getting `defconfig (arm64, None)` in the report.  Can we put the build environment string there instead, as that's supposed to be always set to a meaningful value?

Reply:
This worked before Christmas, have we stopped parsing the compiler info in the backend?

Reply:
That said, I'm happy to use build-environment

Reply:
There's nothing to guarantee the `compiler_version_ext` field will be always in the build meta-data. I ran this locally with build.py and I can see `compiler_version_full` populated in the DB but not the `_ext` one.

Reply:
This patches works for me, using `build_environment` instead: https://termbin.com/fzs6

Reply:
Ok, I'm testing locally with build_environment.
As we're requiring that we can guarantee it will be there.